<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WChatBotMainUIClass</class>
 <widget class="QWidget" name="WChatBotMainUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>121445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WChatBotMainUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 12px &quot;微软雅黑&quot;;
}

QPushButton{
	border:none;
	background-color: rgba(255, 255, 255,0);
	background-position: center;
    background-repeat: no-repeat;
}

QPushButton#pb_left_icon{
	background-image: url(:/QtTest/icon/Setting/chatbot/left.png);
}

QPushButton#pb_open[open=true]{
	background-image: url(:/QtTest/icon/Setting/open.png);
}

QPushButton#pb_open[open=false]{
	background-image: url(:/QtTest/icon/Setting/off.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pb_left_icon">
        <property name="minimumSize">
         <size>
          <width>13</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>13</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_left">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255,0);
font: 24px &quot;微软雅黑&quot;;
color: qlineargradient(spread:pad, x1:0, y1:0, x2:111, y2:20, stop:0 #0066FF, stop:1 #BD00FF);
</string>
        </property>
        <property name="text">
         <string>ChatBot</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>427</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="pb_open">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox{
	font: 20px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}

QCheckBox::indicator {width: 30px; height: 30px;}

QCheckBox::indicator:unchecked {image: url(:/QtTest/icon/Setting/off.png);}

QCheckBox::indicator:checked {image: url(:/QtTest/icon/Setting/open.png);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Activated</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>12133</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_left">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
border-right:1px solid #505050;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="WNavbarButton" name="pb_lib">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="WNavbarButton" name="pb_history">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>370</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="WLibarary" name="lib_page"/>
        <widget class="WHistoryPage" name="history_page"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WNavbarButton</class>
   <extends>QPushButton</extends>
   <header>widget/model/WNavbarButton.h</header>
  </customwidget>
  <customwidget>
   <class>WLibarary</class>
   <extends>QWidget</extends>
   <header>WLibarary.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WHistoryPage</class>
   <extends>QWidget</extends>
   <header>WHistoryPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../AiSound.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pb_left</sender>
   <signal>clicked()</signal>
   <receiver>WChatBotMainUIClass</receiver>
   <slot>slot_back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_left_icon</sender>
   <signal>clicked()</signal>
   <receiver>WChatBotMainUIClass</receiver>
   <slot>slot_back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_back()</slot>
 </slots>
</ui>
